packages/contracts/deploy/00_managing-dao/01_managing-dao_conclude.ts

```diff
@@ -0,0 +1,14 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {EHRE} from '../../utils/types';
+
+const func: DeployFunction = async function (hre: EHRE) {
+  console.log(`\nConcluding ManagingDao deployment.`);
+
+  const {deployments} = hre;
+
+  hre.aragonToVerfiyContracts.push(await deployments.get('DAO_Implementation'));
+  hre.aragonToVerfiyContracts.push(await deployments.get('DAO_Proxy'));
```

Comment:
deployments.get() returns type of `Deployment` and not `AragonVerifyEntry`, should we use `await deployments.get('DAO_Implementation') as AragonVerifyEntry` or something similar

Reply:
`AragonVerifyEntry` needs the properties `address` and `args`. `Deployment` has these 2 parameters (and many more) thus it is compatible and we can leave it as it is